autogen/agentchat/conversable_agent.py

```diff
@@ -134,7 +141,7 @@ def __init__(
             self.llm_config = self.DEFAULT_CONFIG.copy()
             if isinstance(llm_config, dict):
                 self.llm_config.update(llm_config)
-            self.client = OpenAIWrapper(**self.llm_config)
+            self.client = OpenAIWrapper(**self.llm_config, websocket=self.websocket)
```

Comment:
It is a great idea to hook it up directly to the model client. Though this is only one possible reply that could be triggered. This will not write messages produced by tool calls, function calls, and any user-defined reply functions that does not produces message from `self.client`. 

While in many cases streaming LLM messages is exactly what people want, we can make it clear that this doesn't stream other messages. If you also want to cover all other message, a good place to look at is the `generate_reply` function where the reply message got finalized.

Reply:
@ekzhu thank you for the compliment.  For my application, I added the websocket to just this section, because our frontend is client-facing, and we don't want them to see the function calls and code execution.  I will look at the generate_reply function too, thank you.